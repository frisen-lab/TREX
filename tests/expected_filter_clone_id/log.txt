Reading cloneIDs from chrTomato-N:695-724 in tests/data/outs/possorted_genome_bam.bam
Found 3857 reads with usable cloneIDs and UMIs. Skipped 1692 without cell id, 3 without UMI.
Read 3857 reads containing (parts of) the cloneID (2703 full cloneIDs, 71 unique)
Detected 3800 molecules (2676 full cloneIDs, 70 unique)
3639 remain after low-complexity filtering
After cloneID correction, 39 unique cloneIDs remain
After filtering cloneIDs, 37 unique cloneIDs remain
Detected 1808 cells
Found 85 single-read cloneIDs
689 filtered cells remain
Removing 0 bridges from the graph
Removing 0 doublets from the graph (first round)
Removing 0 bridges from the graph (second round)
Removing 0 doublets from the graph (second round)
Detected 10 clones
Clone size histogram
 size count
    1     8
    2     1
  679     1
