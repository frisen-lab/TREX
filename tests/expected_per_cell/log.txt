Trex 0.4.dev253+g7d94d1c.d20231017
Command line arguments: run10x --per-cell --delete -s 695 -e 724 -o trex_per_cell tests/data
Reading cloneIDs from chrTomato-N:695-724 in tests/data/outs/possorted_genome_bam.bam
Found 3857 reads with usable cloneIDs and UMIs. Skipped 1692 without cell id, 3 without UMI.
Read 3857 reads containing (parts of) the cloneID (2703 full cloneIDs, 71 unique)
Detected 3800 molecules (2676 full cloneIDs, 70 unique)
3639 remain after low-complexity filtering
After cloneID correction, 39 unique cloneIDs remain
Detected 1835 cells
Found 92 single-read cloneIDs
697 filtered cells remain
Removing 0 bridges from the graph
Removing 0 doublets from the graph (first round)
Removing 0 bridges from the graph (second round)
Removing 0 doublets from the graph (second round)
Detected 11 clones
Clone size histogram
 size count
    1     8
    2     1
    8     1
  679     1
