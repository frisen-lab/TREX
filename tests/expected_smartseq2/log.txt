Reading cloneIDs from EGFP-30N:2330-2359 in tests/data/smartseq2_test.bam
Found 279 reads with usable cloneIDs. Skipped 0 without cell id, 7 with deletions.
Read 279 reads containing (parts of) the cloneID (135 full cloneIDs, 115 unique)
After cloneID correction, 110 unique cloneIDs remain
Detected 91 cells
91 filtered cells remain
Writing read matrix
Removing 0 bridges from the graph
Detected 89 clones
Clone size histogram
 size count
    1    87
    2     2
