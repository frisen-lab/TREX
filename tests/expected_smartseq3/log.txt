Trex 0.4.dev161+g17ccfe3.d20230601
Command line arguments: smartseq3 --delete --output trex_smartseq3_run --umi-matrix -s 2330 -e 2359 tests/data/smartseq3_test.bam
Reading cloneIDs from EGFP-30N:2330-2359 in tests/data/smartseq3_test.bam
Found 9631 reads with usable cloneIDs and UMIs. Skipped 0 without cell id, 24435 without UMI.
Read 9631 reads containing (parts of) the cloneID (4593 full cloneIDs, 943 unique)
Detected 3448 molecules (2183 full cloneIDs, 779 unique)
After cloneID correction, 424 unique cloneIDs remain
Detected 1528 cells
Found 680 single-read cloneIDs
613 filtered cells remain
Writing UMI matrix
Removing 7 bridges from the graph
Detected 323 clones
Clone size histogram
 size count
    1   233
    2    36
    3    19
    4     7
    5     7
    6     4
    7     9
    8     1
    9     2
   11     1
   12     1
   14     1
   15     1
   23     1
