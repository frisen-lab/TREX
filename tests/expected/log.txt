Trex 0.4.dev150+g2e002b0.d20220427
Command line arguments: run10x --delete --loom --umi-matrix -s 695 -e 724 tests/data/
Reading cloneIDs from chrTomato-N:695-724 in tests/data/outs/possorted_genome_bam.bam
Found 3857 reads with usable cloneIDs and UMIs. Skipped 1692 without cell id, 3 without UMI.
Read 3857 reads containing (parts of) the cloneID (2703 full cloneIDs, 71 unique)
Detected 3800 molecules (2676 full cloneIDs, 70 unique)
After cloneID correction, 6 unique cloneIDs remain
Detected 1921 cells
Found 41 single-read cloneIDs
763 filtered cells remain
Writing UMI matrix
Removing 0 bridges from the graph
Detected 4 clones
Clone size histogram
 size count
    1     1
    2     1
    8     1
  752     1
